yourgov/components/StaffMembersTable/StaffMembersTable.tsx

```diff
@@ -0,0 +1,88 @@
+import { RefObject } from 'react';
+import { Stack } from '@ag.ds-next/react/src/stack';
+import { Button } from '@ag.ds-next/react/src/button';
+import { useTernaryState } from '@ag.ds-next/react/src/core';
+import { FilterIcon } from '@ag.ds-next/react/src/icon';
+import { Box } from '@ag.ds-next/react/src/box';
+import { DataProvider, SortAndFilterProvider } from './lib/contexts';
+import { useSortAndFilter } from './lib/useSortAndFilter';
+import { useFetchData } from './lib/useFetchData';
+import { ActiveFilters } from './ActiveFilters';
+import { SortBySelect } from './SortBySelect';
+import { DataTable } from './DataTable';
+import { DashboardFilterDrawer } from './DashboardFilterDrawer';
+import { FilterSearchInput } from './FilterSearchInput';
+import { FilterBar, FilterBarGroup, FilterRegion } from './FilterBar';
+import { DashboardPagination } from './DashboardPagination';
+import { DataTableSelectAllCheckbox } from './DataTableSelectAllCheckbox';
+import { FilterSwitchInput } from './FilterSwitchInput';
+
+type StaffMembersTableProps = {
+	selectable?: boolean;
+	tableRef?: RefObject<HTMLTableElement>;
+};
+
+const headingId = 'table-heading';
```

Comment:
This needs updating since you updated the H2 in the page
```suggestion
const headingId = 'staff-members-heading';
```